cmake/Modules/FindIrrlicht.cmake

```diff
@@ -13,7 +14,7 @@ if(NOT IRRLICHT_SOURCE_DIR STREQUAL "")
 
 	if(WIN32)
 		if(MSVC)
-			set(IRRLICHT_SOURCE_DIR_LIBS "${IRRLICHT_SOURCE_DIR}/lib/Win32-visualstudio")
+			set(IRRLICHT_SOURCE_DIR_LIBS "${IRRLICHT_SOURCE_DIR}/lib/Win32-VisualStudio")
```

Comment:
Oh, I changed it because it's called `VisualStudio` below, although it shouldn't matter since NTFS is case-insensitive by default.